30-helm/s3-sidecar/pd-archive-backup-to-s3.yaml

```diff
@@ -12,6 +12,20 @@
 # Your particular implementation will depend on the desired storage
 # and the cloud environment (S3, etc.).
 configMaps:
+  pingdirectory-fetch:
+    data:
+      fetch.sh: |-
+        #!/bin/sh
+        set -e
+
+        # This script assumes it is only being run on a single pod of the topology
+        # Demonstration only - no testing or error checking is performed
+
+        # Run the fetch from archive command
+        # Assumes credentials are set as environment variables or that the Pod 
+        # has an associated IAM role that allows it to write to the S3 bucket
+        # userRoot is hardcoded as it is the only backend used in this example
+        aws s3 sync s3://<your-s3-bucket-name>/<path>/userRoot/ /opt/pingdirectory-backup/userRoot
```

Comment:
Should we pull the backups to a different folder? Is there a chance that we could have conflicting files?

Reply:
done - using /tmp/restore as in the restore.sh script